### ISSUE TITLE
Persisting the online toggle should not use the query params

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/pulfalight#1290]
## Summary

When clicking the online toggle it adds a query parameter when you click an item in the table of contents, that looks like this: https://findingaids.princeton.edu/catalog/AC122_c1081?onlineToggle=true

Archivists often copy and paste that URL for patrons and frequently have to remove the parameter, as they don't want that filter to apply to the person they're giving it to. If there's not online content, and it's there, it may be confusing for them to see it.

## Acceptance Criteria

- [ ] Clicking the toggle filters the page to online resources only.
- [ ] Clicking an element after toggling leaves the table of contents toggled, but doesn't add anything to the URL.
- [ ] Opening another collection should not have the toggle set.
- [ ] Opening this collection in another tab should have the toggle set (not a blocker, we just think this is easier, if it's not, don't do that.)

## First Step

Add a test.

## Technical Tips

https://github.com/pulibrary/pulfalight/pull/1295 - This has a PR that uses cookies instead, will help the implementation here.

**Comment by ccleeton** [comment]
We have had some feedback about the toggle while staff like the persistent nature it is displaying in the URL 
https://findingaids.princeton.edu/catalog/MC076_c00001?onlineToggle=true
Is there a way that it could not textually display in the URL? 
Staff do copy and paste the URLs and compids frequently and yes we can just delete it but wondering if there are other options.

**Comment by sdellis** [comment]
[This PR provides an alternate solution to the sticky toggle](https://github.com/pulibrary/pulfalight/pull/1295) that does not appear in the URL. This could be reopened, rebased, and merged today. However, we would want to first roll-back the URL-param-based solution that was initially implemented.

**Comment by sdellis** [comment]
Discussion on 4/25 led us to determine that there is not really a use case for forcing "digital only" via the URL. We don't have to remove that feature necessarily, but it would not be advertised in any way.

**Comment by tpendragon** [comment]
> Discussion on 4/25 led us to determine that there is not really a use case for forcing "digital only" via the URL. We don't have to remove that feature necessarily, but it would not be advertised in any way.

Yeah this was an implementation detail - we wanted clicking a new table of contents item to keep the toggle state, but when going to a new EAD we didn't want it to default to that.
